tgui/packages/tgui/interfaces/Sleeper.js

```diff
@@ -236,6 +236,29 @@ const SleeperChemicals = (props, context) => {
   } = data;
   return (
     <Section title="Occupant Chemicals" flexGrow="1">
+      {
+        (!!occupant && occupant["crisis"])
+      && (
+        <Box
+          style={{ "position": "absolute",
```

Comment:
Not a massive fan of position:absolute. This needs to be checked to see if it dynamically changes correctly when resizing the window.